tests/ImportLoggerTest.php

```diff
@@ -1,91 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Shaarli\NetscapeBookmarkParser;
-
-use Katzgrau\KLogger\Logger;
-use Psr\Log\LogLevel;
-
-/**
- * Class ImportLoggerTest
- *
- * Make sure that the log file is correctly generated.
- */
-class ImportLoggerTest extends TestCase
```

Comment:
Open question: would it makes sense here to check a Mock object here instead of removing the test code entirely?

Reply:
No, I don't think it's necessary ; PSR provides `NullLogger` mock by default in the constructor. Besides there is no point in testing code we don't own.